.github/workflows/build.yaml

```diff
@@ -1,11 +1,13 @@
 name: build
 
 on:
-  push:
-    branches:
-      - "branch-*"
-    tags:
-      - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
+  # TODO: uncomment this condition when ready to start publishing
+  #       packages from this repo
+  # push:
+  #   branches:
+  #     - "branch-*"
+  #   tags:
+  #     - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
```

Comment:
It isn't safe to publish nightly `nx-cugraph` packages from here until we stop publishing them from https://github.com/rapidsai/cugraph (and delete the old ones from the 24.12 dev cycle, as we discussed offline).